sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-spring-security-resource-server/README.md

```diff
@@ -0,0 +1,109 @@
+# OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server library for Java
+
+## Key concepts
+This sample illustrates how to protect an Java web API by restricting access to its resources to authorized accounts only.
+Obtain Bearer Token in the request head of resources and carry out analysis and verification. After verification,
+injects AzureOAuth2AuthenticatedPrincipal object that is associated with the thread of the current user request. 
+Token claimsset, Authorities, JWS headers etc. are accessible from the object which can be used for role based authorization.
```

Comment:
I feel a little confusing by the sentence here at first.

Reply:
I'm going to modify it.